src/plugins/push_v5.js

```diff
@@ -62,6 +62,19 @@ angular.module('ngCordova.plugins.push_v5', [])
           }, number);
         }
         return q.promise;
+      },
+      finish: function(){
+        var q = $q.defer();
+        if (push === undefined) {
```

Comment:
Shouldn't this just check if `push` is falsy? Since it's consistent with the rest of the file I'm sure it's fine, but just a suggestion.